src/script/auth/component/AccountForm.tsx

```diff
@@ -39,77 +40,89 @@ import {RootState, bindActionCreators} from '../module/reducer';
 import * as AuthSelector from '../module/selector/AuthSelector';
 import * as AccentColor from '../util/AccentColor';
 
+const logger = getLogger('AccountForm');
+
 interface Props extends React.HTMLProps<HTMLFormElement> {
   beforeSubmit?: () => Promise<void>;
-  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+  onSubmit: () => void;
   submitText?: string;
 }
 
 const AccountFormComponent = ({account, ...props}: Props & ConnectedProps & DispatchProps) => {
   const {formatMessage: _} = useIntl();
-  const [registrationData, setRegistrationData] = React.useState({
+
+  const [registrationData, setRegistrationData] = useState({
     accent_id: AccentColor.random().id,
-    email: account.email || '',
-    name: account.name || '',
-    password: account.password || '',
-    termsAccepted: account.termsAccepted || false,
+    email: '',
+    name: '',
+    password: '',
+    termsAccepted: false,
```

Comment:
I think there are scenarios where the user data is already filled in, we should probably not change this

Reply:
oohh right